packages/legend-manual-tests/src/__tests__/roundtrip-grammar/cases/DSL_ExecutionEnvironment.pure

```diff
@@ -0,0 +1,15 @@
+###Pure
+Class my::Person
+{
+  givenNames: String[*];
+}
+
+
+###Mapping
+Mapping my::map
+(
+  *my::Person: Pure
+  {
+    givenNames: 'name'
+  }
+)
```

Comment:
can we add proper test here?

Reply:
please read the TODO

Reply:
:) but then because we don't have roundtrip test, we missed several things such as the alphabetical order in serialization schema 🤷

Reply:
yeah but i want to merge this before you do another huge refactor